Azure/azure-cli
# [SQL] Add Server Trust Group commands

**Description**

This PR adds Server Trust Group commands

**Testing Guide**

Create a Server Trust Group with specified resource ids of its members.
``` az sql stg create -g resourcegroup -l location -server-trust-group-name stg-name --trust-scope GlobalTransactions -gm $mi1-id $mi2-id```
Retrieve a Server Trust Group.
 ```az sql stg show -g resourcegroup -l location -server-trust-group-name stg-name```
Delete a Server Trust Group.
```az sql stg delete -g resourcegroup -l location -server-trust-group-name stg-name```
Retrieve a list of Server Trust Groups by instance.
```az sql stg list -g resourcegroup --instance-name mi1-name```
Retrieve a list of Server Trust Groups by location.
```az sql stg list -g resourcegroup -l location```


This checklist is used to make sure that common guidelines for a pull request are followed.
